### ISSUE TITLE
Error in LogCat

### ISSUE THREAD

**Issue by Joyliu290** [benawad/Munchy#1]
Hi Ben, I watched your Youtube Video for this app and I tried to re-create my own version of it. I followed your instruction of the asyntask and making the images and stuff appear on the screen. For a while my app was fine, but now it's giving me warnings (see attached picture of the logcat), and there is a constant "progressBar" on the screen rotating without display anything from yelp.

 W/art: Failed execv(/system/bin/dex2oat --runtime-arg -classpath --runtime-arg  --debuggable --instruction-set=arm --instruction-set-features=smp,-div,-atomic_ldrd_strd --runtime-arg -Xrelocate --boot-image=/system/framework/boot.art --runtime-arg -Xms64m --runtime-a

The logcat does display "V/Business...." but it is not outputting anything besides the progressBar.

Please help and Thank You!

**Comment by benawad** [comment]
I'd love to help you figure this out. It's hard to tell from the logcat output what the problem is. It will be easier to look at your code. I'm assuming this is it https://github.com/Joyliu290/FoodDecisionApp  . I'll take a look at that and let you know what I find.

**Comment by benawad** [comment]
What is the expected behavior?

I ran your app on my phone (Nexus 5x) and with a Genymotion emulator with no errors. It looked like it worked (check out the screenshots below to confirm). 

Are you using the default Android emulator to test?

![p1](https://cloud.githubusercontent.com/assets/7872329/26528291/17f640b2-436e-11e7-9f5f-68c2eba870a6.png)
![p2](https://cloud.githubusercontent.com/assets/7872329/26528293/1e3fb21e-436e-11e7-8289-341802ec55f5.png)

**Comment by Joyliu290** [comment]
Hi Ben, I figured out the problem, and it is that Yelp for whatever reason has blocked my home wifi IP address, and my access to yelp's pages if I use my home wifi. That is really weird. Do you have any suggestions to get away from this problem?

**Comment by benawad** [comment]
That is weird, the only way I can think to get around that is to use a vpn.

**Comment by Joyliu290** [comment]
Hey Ben, just a heads up since I've watched your tutorial on how to integrate Yelp API...I've contacted Yelp API support people on how to resolve my issue and they said that they strictly prohibit any sort of scraping of data. In your tutorial, if I'm not mistaken, the way that you were able to retrieve the images is by using Jsoup scraping. I guess the fact that Yelp hasn't blocked your IP address is a good sign, but it's just something that I thought I'd like to bring it up to your attention as it is something you mentioned in your tutorial and it may harm others who watch your tutorial.
